Mathlib/RingTheory/Valuation/ValuativeRel/Basic.lean

```diff
@@ -172,14 +184,14 @@ def posSubmonoid : Submonoid R where
   mul_mem' := zero_srel_mul
   one_mem' := zero_srel_one
 
+@[simp] lemma zero_srel_posSubmonoid_def (x : posSubmonoid R) : 0 <ᵥ x.val := x.prop
```

Comment:
```suggestion
@[simp] lemma zero_srel_of_posSubmonoid (x : posSubmonoid R) : 0 <ᵥ x.val := x.prop
```
`posSubmonoid` is the type of an argument so it should be preceded by `of`. And I don't think `def` shouldn't be here.

Reply:
I think you're wrong about the `of` though. The lemma name comes from `0 srel (_ : posSubmonoid)`, not `(_ : posSubmonoid) : 0 srel _`

Reply:
Either is fine to me. But I assume it should be `val` and not `coe`?